package.json

```diff
@@ -12,7 +12,7 @@
   "scripts": {
     "boot:mocha": "nodemon --exec 'mocha -R min'",
     "boot:mocks": "http-server ./mocks &",
-    "test": "yarn boot:mocks && yarn boot:mocha"
+    "test": "http-server ./mocks & mocha \"./test/**/*.js\" --timeout 10000"
```

Comment:
Every now and then, a test seemed to time out. Setting the flag `--timeout` increases the default timeout to 10 seconds.